.github/workflows/stale-pull-requests.yml

```diff
@@ -0,0 +1,39 @@
+name: Check stale pull requests
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 12 * * *'
+
+permissions:
+  issues: read
+  pull-requests: write
+
+env:
+  BEFORE_STALE: 14
+  BEFORE_CLOSE: 7
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v5
```

Comment:
The action is producing this warning:
```
Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
```

Do they have a newer version that does not use `set-output` already?

Reply:
Yes, they released v6 some days ago which already upgrade to the new format: https://github.com/actions/stale/pull/839